public/app/views/repositories/_badge.html.erb

```diff
@@ -16,9 +16,7 @@
      <% when 'subject' %>
       <i class="fa fa-tags fa-4x"></i>
      <% when 'agent' %>
-     <span class="fa-stack fa-2x">
-       <i class="fa fa-institution fa-stack-2x"></i>
-       <i class="fa fa-users fa-stack-1x fa-inverse"></i>
+      <i class="fa fa-users fa-4x"></i>
      </span>
```

Comment:
@Blake- I think the closing span tag was missed - it should be removed.

Reply:
@lmcglohon D'oh!